src/Estina/SmartFocus/Api/Rest/Notification.php

```diff
@@ -50,4 +56,116 @@ public function send(
 
         return $response;
     }
+
+    /**
+     * Send batch of email transactions at once.
+     *
+     * Also supports sending of dynamic content.
+     *
+     * @param SimpleXMLElement $xmlObject The xml object to post.
+     *
+     * @return string|false XML response or FALSE on failure
+     */
+    public function post(SimpleXMLElement $xmlObject)
```

Comment:
Hey thanks for having a look. The sendObjects call uses the SOAP API to perform a similar operation - this call though is not the same. Have a look here - the endpoint is different as well https://help-developer.smartfocus.com/#NMP_TRIGGER/Getting-Started-NMP.htm?Highlight=NMSXML.